src/Installer.php

```diff
@@ -218,6 +275,49 @@ private function installPuli()
     }
 
     /**
+     * Validates if the passed version is valid.
+     *
+     * @param string $version  The passed version.
+     * @param array  $versions The available versions.
+     *
+     * @return bool Whether the version is valid.
+     */
+    private function validateVersion($version, array $versions)
+    {
+        return in_array($version, $versions);
```

Comment:
I would add the `true` argument to enable strict checks and inline this method. There's no real benefit in having a separate method here, not even readability IMO.